cluster/ack.go

```diff
@@ -130,13 +132,24 @@ func (c *ACKCluster) SetTTL(ttl time.Duration) {
 func (c *ACKCluster) ListNodeNames() (pkgCommon.NodeNames, error) {
 	essClient, err := c.GetAlibabaESSClient(nil)
 	if err != nil {
-		return nil, err
+		return nil, emperror.Wrap(err, "failed to get Alibaba ESS client")
```

Comment:
This log message is duplicated, we already put this message to the log inside cGetAlibabaESSClient